devsetup/Makefile

```diff
@@ -129,7 +130,7 @@ edpm_compute_repos: ## Configure EDPM compute VM with repositories
 	scripts/edpm-compute-repos.sh ${EDPM_COMPUTE_SUFFIX}
 
 .PHONY: edpm_compute_cleanup
-edpm_compute_cleanup: ## Delete EDPM compute VM
+edpm_compute_cleanup: ansibleee_key_cleanup ## Delete EDPM compute VM
```

Comment:
in a scenario you need to delete just one node it would break, as it would remove the secret for the other nodes

Reply:
The makefile doesn't really support selective node cleanup at this moment. The only way to do that would be to manually set `EDPM_TOTAL_NODES` != 1 and `EDPM_COMPUTE_SUFFIX` to your chosen nodes index. But that is very unwieldy, and if you were then to remove rest of the nodes, you would have to apply the same procedure again and again.

Reply:
`make edpm_compute_cleanup EDPM_COMPUTE_SUFFIX=3` would delete just the edpm-compute-3 node